File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/commands/RowBasedCommand_ViewModelMyFlagsRow.cs
/// <filename>
///     RowBasedCommand_ViewModelMyFlagsRow.cs
/// </filename>


namespace commands
{
    public class RowBasedCommand_ViewModelMyFlagsRow
  {
        private bool isCheckboxesCheckBoxChecked;
     
        public     virtual bool getIsCheckboxesCheckBoxChecked()
    {
      return this.isCheckboxesCheckBoxChecked;
    }
     
        private int RowIndex;
     
        public     virtual int getRowIndex()
    {
      return this.RowIndex;
    }
  }
}

File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/commands/RowBasedCommand_ViewModelMyFlagsRow.cpp

#include "RowBasedCommand_ViewModelMyFlagsRow.h"

namespace commands
{
  bool RowBasedCommand_ViewModelMyFlagsRow::getIsCheckboxesCheckBoxChecked( ) 
  {
    return this->isCheckboxesCheckBoxChecked;
  }
  int RowBasedCommand_ViewModelMyFlagsRow::getRowIndex( ) 
  {
    return this->RowIndex;
  }
}

